### ISSUE TITLE
Add support to brackets on cols value

### ISSUE THREAD

**Issue by VitorLuizC** [VitorLuizC/vue-data-tablee#13]
Using brackets on cols value throws error on `get` helper.

```js
[
  { label: 'Company', value: 'users[0].company.name' }
]
```

I've discovered some ways to do it and a path module alternative. I'd like to use it or something like IDX (https://github.com/facebookincubator/idx).

```js
function getProperty (object: object, property: string): any {
  try {
    const get = new Function('object', `return object.${property}`);
    return get(object);
  } catch (_) {
    return;
  }
}
```

**Comment by VitorLuizC** [comment]
I'll close after publish it.

**Comment by VitorLuizC** [comment]
Published on 0.13.0.
